### DISCUSSION TITLE
Access List problem: nginx shows router IP rather than client's public IP

### DISCUSSION THREAD

**Discussion by hrmtc** [NginxProxyManager/nginx-proxy-manager#2277] [Q&A]
Hi everyone.

I'm trying to figure it out why my access list aren't working to limit access from WAN side.

When I look on access.log I can see that nginx is registering the client IP of my router rather than the client IP that originated the connection from outside.

This is normal behaviour?

Could someone confirm this please?

Thanks!

**proxy-host-11_access.log**

[`20/Sep/2022:14:38:58 -0300] - 200 200 - GET https [www.website.com](https://www.website.com/) "/" **[Client [192.168.1.1]**(https://192.168.11.1/)] [Length 5074] [Gzip -] [Sent-to [192.168.1.31](https://192.168.1.31/)] "Mozilla/5.0 (iPhone; CPU iPhone OS 16_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like $`

**Comment by igisho** [comment]
Same here.
